recipes/jenkins.rb

```diff
@@ -249,8 +256,14 @@ def jenkins = Jenkins.getInstance()
       --domain #{server_name}
       --fullchain-file #{cert_path}
       --key-file #{key_path}
+      --reloadcmd /root/cert-update-hook.sh
+      --force
```

Comment:
The `--force` flag needs to be added in order to force a change to a deployed, up-to-date certificate.